### PR TITLE
Fixing builds and start command

### PR DESCRIPTION
**Pull request by roborourke**
Temporary resolution for #115 and #114

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
-Tested up to: 6.2  
-Requires PHP: 7.2  
-License: GPL v3 or later  
-Contributors: johnbillion, humanmade  
+Stable tag: 0.2.17
+Requires at least: 5.4
+Tested up to: 6.2
+Requires PHP: 7.2
+License: GPL v3 or later
+Contributors: johnbillion, humanmade
```
Comment (johnbillion) [inline README.md]: These trailing spaces need to be retained, they're line breaks

Comment (roborourke) [reply README.md]: I think that'll be my editor set to remove them automatically, but that's weird no? Why would spaces at the end of a line be line breaks?

Comment (roborourke) [inline README.md]:
```suggestion
Stable tag: 0.2.17  
Requires at least: 5.4  
Tested up to: 6.2  
Requires PHP: 7.2  
License: GPL v3 or later  
Contributors: johnbillion, humanmade  
```

Comment (johnbillion) [reply README.md]: Markdown `¯\_(ツ)_/¯`

Comment (johnbillion) [reply README.md]: https://github.com/humanmade/authorship/blob/develop/.editorconfig#L14-L15

File: README.md [hunk 2]
```diff
@@ -61,9 +61,10 @@ _Features without a checkmark are still work in progress._
 ### For development use
 
 * Clone this repo into your plugins directory
-* Install the dependencies:  
+* Ensure you have Composer v2 and Node v16 installed
```
Comment (tomjn) [inline README.md:64]: Would it make more sense to explicitly mention `nvm use`?

Comment (roborourke) [reply README.md:64]: @tomjn I'm going to update it all soon anyway, the specific tool people use to manage node versions is up to them

**Review by johnbillion — APPROVED** [review]
All looks good. This reminds me that #25 would be great to get done too.

**Comment by roborourke** [comment]
@johnbillion for #25 I think the e2e stuff provided by wp-scripts is the way to go.

**Pull request merged**